config/webpack.dev.js

```diff
@@ -26,7 +28,7 @@ const METADATA = webpackMerge(commonConfig.metadata, {
   HMR: HMR
 });
 
-const frontendConfig = commonConfig.frontendConfig;
+const frontendConfig = _.merge(commonConfig.frontendConfig, { apiEndpoint: 'http://localhost:8080/v1' });
```

Comment:
So you did end up deciding that the Webpack configuration was the best way to use env-specific settings? I don't mind moving all globals from Angular to Webpack if you'd prefer. I believe it was set up that way before.